cli/src/main/scala/aqua/AquaCli.scala

```diff
@@ -77,12 +78,13 @@ object AquaCli extends IOApp with Logging {
         // otherwise continue program execution
         h.map(_ => helpAndExit) orElse v.map(_ => versionAndExit) getOrElse {
           val target =
-            if (toAir) AirTarget
+            if (isScriptStorage) AirTarget
+            else if (toAir) AirTarget
             else if (toJs) JavaScriptTarget
             else TypescriptTarget
           val bc = {
-            val bc = TransformConfig(wrapWithXor = !noXor, constants = constants)
-            bc.copy(relayVarName = bc.relayVarName.filterNot(_ => noRelay))
+            val bc = TransformConfig(wrapWithXor = !(noXor || isScriptStorage), constants = constants)
+            bc.copy(relayVarName = bc.relayVarName.filterNot(_ => noRelay || isScriptStorage))
```

Comment:
Maybe it's better to do like this:
```
if (isScriptStorage) {
  toAir = true
  noXor = true
  noRelay = true
}
```

This way, it's clear that `--script` is the same as `--no-xor --no-relay --air`